diesel/src/pg/expression/expression_methods.rs

```diff
@@ -1302,8 +1302,129 @@ pub trait PgJsonbExpressionMethods: Expression + Sized {
     {
         Grouped(HasKeyJsonb::new(self, other.as_expression()))
     }
+    /// Creates a PostgreSQL `?|` expression.
+    ///
+    /// This operator checks if any of the strings in the right hand side array exists as top level key in the given JSONB
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # include!("../../doctest_setup.rs");
+    /// #
+    /// # table! {
+    /// #    contacts {
+    /// #        id -> Integer,
+    /// #        name -> VarChar,
+    /// #        address -> Jsonb,
+    /// #    }
+    /// # }
+    /// #
+    /// # fn main() {
+    /// #     run_test().unwrap();
+    /// # }
+    ///
+    /// # #[cfg(feature = "serde_json")]
+    /// # fn run_test() -> QueryResult<()> {
+    /// #     use self::contacts::dsl::*;
+    /// #     let conn = &mut establish_connection();
+    /// #     conn.execute("DROP TABLE IF EXISTS contacts").unwrap();
+    /// #     conn.execute("CREATE TABLE contacts (
+    /// #         id SERIAL PRIMARY KEY,
+    /// #         name VARCHAR NOT NULL,
+    /// #         address JSONB NOT NULL
+    /// #     )").unwrap();
+    /// #
+    /// let santas_address: serde_json::Value = serde_json::json!({
+    ///     "street": "Article Circle Expressway 1",
+    ///     "city": "North Pole",
+    ///     "postcode": "99705",
+    ///     "state": "Alaska"
+    /// });
+    /// diesel::insert_into(contacts)
+    ///     .values((name.eq("Claus"), address.eq(&santas_address)))
+    ///     .execute(conn)?;
+    ///
+    /// let any_key_exists = contacts.select(address.has_any_key(vec!["street", "city", "rudolf"])).get_result::<bool>(conn)?;
+    /// assert!(any_key_exists);
+    ///
+    /// let santas_with_address_postcode = contacts.select(id).filter(address.has_any_key(vec!["street", "city", "rudolf"])).get_result::<i32>(conn)?;
+    /// assert_eq!(1, santas_with_address_postcode);
+    /// #     Ok(())
+    /// # }
+    /// # #[cfg(not(feature = "serde_json"))]
+    /// # fn run_test() -> QueryResult<()> {
+    /// #     Ok(())
+    /// # }
+    /// ``````
+    fn has_any_key<T>(self, other: T) -> dsl::HasAnyKeyJsonb<Self, T>
+    where
+        T: AsExpression<Array<VarChar>>,
+    {
+        Grouped(HasAnyKeyJsonb::new(self, other.as_expression()))
+    }
+    /// Creates a PostgreSQL `?&` expression.
```

Comment:
Empty line before this please

Reply:
Thanks! There were just a couple of minor format mistakes. They should be fixed now